rotational-cipher/rotational_cipher.py

```diff
@@ -1,2 +1,31 @@
 def rotate(text, key):
-    pass
+    if type(text)!=str:
+        return('first input should be a string')
+    else:
+        text = text.lower()
+        alphabet = 'abcdefghijklmnopqrstuvwxyz'
+        tempList = []
+        count = 0
+        for letter in alphabet:
+            tempList.append((letter,count))
+            count += 1
+        lettersDict = dict(tempList)
+        newText = ""
+        for letter in text:
+            if letter==" ":
+                newText += letter
+            else:
+                newKey = lettersDict.get(letter)
+                if newKey + key <= 25:
+                    newKey += key
+                else:
+                    newKey = key - (25 - newKey) - 1
```

Comment:
Are you trying to deal with key > 25, but what if they are larger than 50? Do you think "key%25" can solve this and yield newKey always in [0,25]?